Tests/test_file_webp_metadata.py

```diff
@@ -1,7 +1,5 @@
 from io import BytesIO
 
-import pytest
```

Comment:
It's probably not worth fixing lint until we pass the: "Does it work" and "Is it a good idea" tests.